src/RestControllers/FhirPatientRestController.php

```diff
@@ -32,41 +32,42 @@ public function __construct($pid)
     public function post($fhirJson)
     {
         $data = $this->fhirService->parsePatientResource($fhirJson);
-        
+
         $validationResult = $this->patientService->validate($data);
         $validationHandlerResult = RestControllerHelper::validationHandler($validationResult);
         if (is_array($validationHandlerResult)) {
             return $validationHandlerResult; }
```

Comment:
```suggestion
            return $validationHandlerResult;
        }
```

Travis sir will be happy after this 😃